data/mods/mixandmega/scripts.ts

```diff
@@ -95,10 +95,12 @@ export const Scripts: ModdedBattleScriptsData = {
 				species.types = [species.types[0], deltas.type];
 			}
 			const baseStats = species.baseStats;
-			for (const statName in baseStats) {
+			let statName: StatID;
+			for (statName in baseStats) {
 				baseStats[statName] = this.battle.clampIntRange(baseStats[statName] + deltas.baseStats[statName], 1, 255);
 			}
 			species.weighthg = Math.max(1, species.weighthg + deltas.weighthg);
+			// @ts-ignore
```

Comment:
I don't like using `ts-ignore` in places where it was previously used and then removed, I would rather typecast `any` or `unknown` here.

Reply:
I was following the pattern used in other `originalMega` references like this:
https://github.com/smogon/pokemon-showdown/blob/master/config/formats.ts#L781-L782

I feel like the right solution is to add this prop to `Species`

Reply:
No, there’s no need to edit the species class for an OM

Reply:
Casted in latest!